src/checks/hyper-training.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  description: 'A Pokémon can only be hyper trained if level 100.',
+  filter (pkmn) {
+    return pkmn.level !== 100;
+  },
+  field: 'level',
+  test (pkmn) {
+    return !(pkmn.hyperTrainedHP || pkmn.hyperTrainedAtk ||
```

Comment:
Can you change `hyperTrainedHP` to `hyperTrainedHp`? We recently added that as an alias for consistency with the other stuff. (You might need to run `npm install pkparse@latest`.)